versions/strategyMatrix.py

```diff
@@ -70,6 +74,8 @@ def deb_package_details(pkg):
         'targetPackageName': target_package_name,
         'nrPackageUrl':
 f"https://nr-downloads-main.s3.amazonaws.com/infrastructure_agent/linux/apt/pool/main/f/fluent-bit/{target_package_name}",
+        'repoArch': f"{pkg['arch']}",
```

Comment:
Why not simply...
```suggestion
        'repoArch': pkg['arch'],
```
?

Reply:
Without the formatting is not treated as string apparently